src/features/editUserProfile/ui/form.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useStore } from 'effector-react';
+import { Box, TextField, Container, Typography, Button, EditIcon } from 'shared';
+import { viewerModel } from 'entities/viewer';
+import { User } from 'entities/viewer/model/types';
+import { editUserProfileModel } from 'features/editUserProfile';
+import { DeleteUser } from 'features/editUserProfile/ui/deleteUserButton';
+
+export const EditUserProfile: React.FunctionComponent = () => {
+  const user = useStore(viewerModel.$user);
+  const navigate = useNavigate();
+
+  const editUserProfileFilds: { lable: string; key: keyof User }[] = [
+    { lable: 'Nickname', key: 'nickname' },
+    { lable: 'E-mail', key: 'email' },
+    { lable: 'Surname', key: 'surname' },
+    { lable: 'Name', key: 'name' },
+  ];
+
+  return (
+    <Container maxWidth="md">
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          width: 1,
+          height: 'calc(100vh - 65.3px)',
```

Comment:
try not to use floats in pixels, it can be blurry on some devices

Reply:
fixed